test/parse_cmp.rb

```diff
@@ -208,8 +208,7 @@ def plist_to_obj_xml(xml)
     '</real>' => '</f>',
     '<real/>' => '<f/>',
     '<true/>' => '<y/>',
-    '<false/>' => '<n/>',
-  }.each do |pat, rep|
+    '<false/>' => '<n/>', }.each do |pat, rep|
```

Comment:
I prefer the original.

Reply:
Possible options https://docs.rubocop.org/rubocop/cops_layout.html#layoutmultilinehashbracelayout